libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -753,131 +730,6 @@ string ofSystemTextBoxDialog(string question, string text){
 		 DestroyWindow(dialog);
 		 EnableWindow(WindowFromDC(wglGetCurrentDC()), TRUE);
 
-	#else
-
```

Comment:
The code I deleted here is _exactly_ repeats what the code under `#ifdef _MSC_VER` do, except it was targeting chars instead of wide chars.